### ISSUE TITLE
garderobel_api

### ISSUE THREAD

**Issue by MaxJoas** [ezet/glappen-client#1]
I'd like to use the stripe_sdk package for Flutter. In order to get a better understanding for the package. I tried to run this example app. It seems however that I don't have access to garderobel_api. I tried all the links in the pubspec.yaml file.

**Comment by ezet** [comment]
Unfortunately you won't be able to compile your own version of the app against my backend. You would have to deploy your own backend in order to run this. I am working on a simpler demo project for stripe sdk, available here https://github.com/ezet/stripe-sdk/tree/master/example/app

**Comment by MaxJoas** [comment]
Thank's very much for the quick reply I am excited to check it out and use the package.
Just that you know I was actually able to run the app after I cloned this repo https://github.com/ezet/glappen-firebase-client and update a few packages and and deprecated methods.

**Comment by ezet** [comment]
Were you able to log in as well ?

**Comment by MaxJoas** [comment]
Yes I was able to log in then to use the camera to scan something and to add a credit card.
